Makefile

```diff
@@ -104,6 +104,9 @@ devel-install: $(WEBPACK_TEST)
 
 dist-gzip: $(TARFILE)
 
+dist-gzip-dump:
+	echo ./$(TARFILE)
```

Comment:
Not sure why we need this new target, instead of merging it with dist-gzip. But if we have it, then IMHO it should depend on `$(TARFILE)` by itself, so that you don't have to call two targets and run into race conditions with `-j`.